identity-apps-core/features/org.wso2.identity.apps.authentication.portal.server.feature/resources/web.xml.j2

```diff
@@ -88,6 +88,10 @@
        <param-value>{{ authenticationendpoint.account_recovery_endpoint_url }}</param-value>
    </context-param>
    {% endif %}
+    <context-param>
```

Comment:
Shall we add a check for config definition `{% if authenticationendpoint.authenticator_validation_enabled is defined %}`